apps/snitch_core/lib/core/data/model/package.ex

```diff
@@ -36,4 +37,35 @@ defmodule Snitch.Data.Model.Package do
   def get(query_fields) do
     QH.get(Package, query_fields, Repo)
   end
+
+  @doc """
+  Calculate the cost of packages for an order
+  """
+
+  @spec compute_package_total(Order.t()) :: Money.t()
+  def compute_package_total(order) do
+    case get_packages(order) do
+      [] ->
+        MoneyTools.zero!()
+
+      packages ->
+        packages
+        |> Stream.map(&Map.fetch!(&1, :total))
+        |> Enum.reduce(&Money.add!/2)
+    end
+  end
+
+  @doc """
+  Query of packages related to order
+  """
+  @spec get_packages(Order.t()) :: list(Package.t())
+  def get_packages(order) do
```

Comment:
how about `packages_for_order()` as fn name. 

Also can we use use OrderModel.order_packages(order) in this function, to avoid repeatitive changes that might come in future or having only one function either in package or order.

Reply:
You're right, this function might not be needed:
```elixir
packages = Map.fetch!(Repo.preload(order, [:packages]), :packages)
```
Best way to get packages of an order.